### ISSUE TITLE
Apollo Kotlin should still handle ping messages before connection ack but after websocket is established 

### ISSUE THREAD

**Issue by vincentjames501** [apollographql/apollo-kotlin#4544]
### Summary

Frameworks like Hasura send a Ping IMMEDIATELY after a socket has been established. Apollo Kotlin will log this when this happens:

```
unknown graphql-ws message while waiting for connection_ack: 'ping
```

According to the [spec](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#ping): `The Ping message can be sent at any time within the established socket.`

Here is the thread over there where I thought it was a Hasura bug, but after reading the spec more closely I agree that this should be handled by clients.

https://github.com/hasura/graphql-engine/issues/9246

Thankfully this client handles it better than dgs but still annoying that it logs warnings all the time :) 

### Version

3.7.1

### Logs

```
unknown graphql-ws message while waiting for connection_ack: 'ping
```

### Steps to reproduce the behavior

Connect to a framework like Hasura over the `GraphQLWsProtocol` or just simply send a ping to Apollo Kotlin client before connect_ack message.

**Comment by martinbonnin** [comment]
Thanks for sending this! A fix is there: https://github.com/apollographql/apollo-kotlin/pull/4555

**Comment by martinbonnin** [comment]
It's now available in `3.7.2`
